### ISSUE TITLE
forward arguments to read

### ISSUE THREAD

**Issue by visr** [evetion/GeoDataFrames.jl#14]
Since `GeoDataFrames.read` calls `ArchGDAL.read`, it'd be nice to forward, or otherwise make accesible, the keyword arguments to that function: https://yeesian.com/ArchGDAL.jl/dev/reference/#ArchGDAL.read-Tuple{AbstractString}.

**Comment by evetion** [comment]
Do you have a use case here? I think it could be useful, but the arguments that you need to pass are not user friendly.

**Comment by visr** [comment]
The use case was wanting to use `alloweddrivers`, since it seemed to want to read some online GeoPackage as a TopoJSON file. However I cannot reproduce this now. I tried, but all I managed was this cool example of a streaming online read that totally works:

```julia
url = "https://github.com/flatgeobuf/flatgeobuf/blob/master/test/data/countries.fgb?raw=true"
GeoDataFrames.read("/vsicurl_streaming/" * url)
```

Though another good example would be the options needed for the multi_geom example here: https://yeesian.com/ArchGDAL.jl/latest/tables/
```julia
ArchGDAL.read("data/multi_geom.csv", options = ["GEOM_POSSIBLE_NAMES=point,linestring", "KEEP_GEOM_COLUMNS=NO"])
```

> the arguments that you need to pass are not user friendly

For the options I raised https://github.com/yeesian/ArchGDAL.jl/issues/194. If you have other ideas for improvements perhaps they are also best addressed in ArchGDAL?
